### PR TITLE
Provide a really nice user experience

### PR DESCRIPTION
**Pull request by ColinRTaylor**
@dmcgua01 Can you please review??

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
   <head>
     <title>Both Company</title>
     <style>
+      body {
+        background: cornsilk;
+      }
```
Comment (dmcgua01) [inline index.html:7]: There should really be a newline here

**Review by dmcgua01 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ColinRTaylor) [reply index.html:7]: @dmcgua01 This is absurd, I have a STRONG preference for have no line breaks between blocks, please reconsider. Have no newlines will allow us to cram more code into a single file.

Comment (dmcgua01) [reply index.html:7]: more code is better code for both company

**Review by dmcgua01 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (dmcgua01) [reply index.html:7]: how do i approved the Pull Request????

**Pull request merged**